views/breach-detail.hbs

```diff
@@ -1,68 +1,55 @@
-<main id="breach-detail" class="breach-detail container clear-header">
-  <div class="row jst-cntr">
-    <div class="col-9 flx-cntr">
-      <div class="breach-detail-logo-wrapper">
-        <img class="breach-detail-logo breach-logo" alt="{{ featuredBreach.Name }} logo" src="/img/logos/{{ featuredBreach.LogoPath }}" />
-      </div>
-      <h2 class="headline">{{featuredBreach.Name}}</h2>
-      <h3 class="breach-type">{{breachCategory}}</h3>
-    </div>
-  </div>
-</main>
 {{#getBreachDetail}}
-  <section class="bg-white">
 
-    {{#if overview}}
-      <div class="row jst-cntr">
-        <div class="col-9 padding-top-zero">
-          {{> breach-detail-content-group overview}}
-          {{#if delayedReporting }}
-            <a class="text-link" href="#{{delayedReporting.href}}">
-              {{delayedReporting.headline}}
-            </a>
-          {{/if}}
+  <main id="breach-detail" class="breach-detail container clear-header bg-ink-light">
+    <div class="row jst-cntr">
+      <div class="col-9 flx-cntr">
+        <div class="breach-detail-logo-wrapper">
+          <img class="breach-detail-logo breach-logo" alt="{{ breach.Name }} logo" src="/img/logos/{{ breach.LogoPath }}" />
```

Comment:
nit: I think the `alt=""` should probably use `{{ breach.Title }}` (since an alt tag of "Bell2017" may not be super useful).
Or I guess we could argue if we just use an empty alt tag since maybe a screen reader doesn't get much valuable information from reading "Bell (2017 Breach) logo", when the following headline clearly reads "Bell (2017 Breach)".